### ISSUE TITLE
Pointer Masking Example with N=16

### ISSUE THREAD

**Issue by martinmaas** [riscv/riscv-j-extension#51]
Feedback by Ved Shanbhogue:

> Would it be better if the example was representative of the only N
> values (N=16) supported by the extension?

**Comment by SiFiveHolland** [comment]
Also the current example in the spec is wrong. The hex doesn't match the binary:

```
0xAAFFFFFF12345678
NVBITS[1010101] VBITS[11111111111111111111111110001...000]
```

Bit 56 is set, so this is `0xABFFFFFF12345678`.

**Comment by martinmaas** [comment]
Addressed in 92a88a2.
